controller_gains:
    dz_pre_trans: 0.05
    enable_ground: true
    att_pid:
      kpGround: [12.5,12.5,3.5] #10.5 10.5 3.5 
      kp: [8.5, 8.5, 8.5] # 4.5 4.5 3.5   ## 5.5 (full flip)
      ki: [1.0,1.0,1.0] # 1.0 1.0 0.8
      kd: [6.5,6.5,6.5] # 4.5 4.5 2.8  ##7.5 (full flip)

    pos_pid:
      kp: [13.5,13.5,16.0]#[11.5,11.5,16.0]
      ki: [6.0,6.0,8.0] # 0.0 0.0 2.0
      kd: [10.0,10.0,12.0]#[2.5, 2.5,3.5]
    sat:
      ex: 0.5
      ev: 1.0
      iex: 1.0
      ier: 2.0
    weight: 4.1
    Inertia_matrix: [0.001,0.0,0.0,0.0,0.001,0.0,0.0,0.0,0.001]
    Allocation_matrix: [0.301429102388748,0.735955403463547,-0.744949427323837,-0.421971587344518,0.172298872444366,-0.794293521479409,0.819043900300765,0.046510193764713,0.904741735148129,-0.593704394980915,-0.293022583675305,-0.421163826202063,0.953420437122514,0.265376405450306,-0.138753111972226,-0.751244359993357,-0.30096991363634,0.325399347714704,0.599323048268742,-0.802845571061326,-0.247593555310946,0.546509986337043,0.556699796387724,-0.658383409157718,-0.00888400144355125,0.215117903216544,0.206878060048819,-0.241382414880486,-0.0474608747180168,-0.217249131581812,-0.273493529619868,0.221027676501879,0.168915338845869,0.127629220480947,0.0214932509610041,-0.259584331122989,-0.0801275667833885,-0.107326620103627,-0.090396649549601,0.243303861879375,0.438758999209262,-0.333424682483333,0.291021295990369,0.258926401815371,-0.380895378375629,-0.293347679694737,0.402790861728662,-0.189524582517963]
    null_space: [-0.437068550691247,-0.656450882828844,0.009857766752594,-0.379798824331596,0.162556277261114,-0.277037511181494,0.246121959877196,0.264478076385271,0.13308770418296,0.165138935809909,0.466513984795314,0.227584739874,0.564724680145995,0.185942302227035,0.381178112968572,0.4322053043543]
    num_props: 8
    cmd_min: 0.0
    cmd_max: 100
    yaw_moment_limit: 3.0
    rp_moment_limit: 4.0 # roll and pitch moment limit
    
    

